bin/test-parse

```diff
@@ -88,6 +88,15 @@ function main() {
         echo "Unit tests passed!"
     fi
 
+    # Validate Airflow tags
+    # see Github Issue: https://github.com/mozilla/telemetry-airflow/issues/1443
+    python3 bin/test-dag-tags.py
```

Comment:
i would put this in a separate script, and a separate CircleCI test check as well

Reply:
yes, if we decide to go with the parsing approach then I'd move it into it's own step in circleci

Reply:
Ok, I'd say I'll go with the SQL approach. As Jeff mentioned this way we are already interacting with parsed values by Airflow so we can be sure they are set correctly.

For the above reason, I will leave the script execution where it correctly is in order to avoid having to spin up Airflow locally twice in a row (docker-compose up) as it will just extend the time the pipeline takes to complete.